java/client/src/main/java/glide/managers/CommandManager.java

```diff
@@ -392,4 +372,50 @@ private Response exceptionHandler(Throwable e) {
         }
         throw new RuntimeException(e);
     }
+
+    /**
+     * Add the given set of arguments to the output Command.Builder.
+     *
+     * @param arguments The arguments to add to the builder.
+     * @param outputBuilder The builder to populate with arguments.
+     */
+    public static void populateCommandWithArgs(String[] arguments, Command.Builder outputBuilder) {
+        populateCommandWithArgs(
+                Arrays.stream(arguments)
+                        .map(value -> value.getBytes(StandardCharsets.UTF_8))
+                        .collect(Collectors.toList()),
+                outputBuilder);
+    }
+
+    /**
+     * Add the given set of arguments to the output Command.Builder.
+     *
+     * @param arguments The arguments to add to the builder.
+     * @param outputBuilder The builder to populate with arguments.
+     */
+    private static void populateCommandWithArgs(
+            GlideString[] arguments, Command.Builder outputBuilder) {
+        populateCommandWithArgs(
+                Arrays.stream(arguments).map(GlideString::getBytes).collect(Collectors.toList()),
+                outputBuilder);
+    }
+
+    /**
+     * Add the given set of arguments to the output Command.Builder.
```

Comment:
Implementation note would be excellent here, to indicate that if total length of the args (in bytes) exceeds MAX_REQUEST_ARGS_LENGTH, we create a protobuf message with a pointer to a leaked bytes vector.  Otherwise, we add the arguments directly into the message.